plugins/auth/index.js

```diff
@@ -15,7 +15,9 @@ const sugar = require('hapi-auth-cookie');
 const tokenRoute = require('./token');
 const uuid = require('uuid/v4');
 
-const EXPIRES_IN = '2h';
+const EXPIRES_IN = 7200; // 2h
+const MAX_EXPIRES_IN = 43200; // 12h
```

Comment:
It is better to use a calculation formula so that there is no mistake.
https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/auth/index.js#L121

Reply:
That's right. I fixed!